tests/transactional/enable_selinux.pm

```diff
@@ -49,14 +67,21 @@ sub run {
         }
     }
 
-    assert_script_run('selinuxenabled');
-    validate_script_output("getenforce", sub { m/Enforcing/ });
-    validate_script_output("sestatus", sub { m/Current mode:.*enforcing/ });
-    validate_script_output("sestatus", sub { m/Mode from config file:.*enforcing/ });
-    record_info('SELinux', script_output('sestatus'));
-    record_info('Audit report', script_output('aureport'));
-    record_info('Audit denials', script_output('aureport -a', proceed_on_failure => 1));
-    upload_logs($audit_log);
+    check_enforcing;
+
+    # disable and re-enable SELinux
+    assert_script_run "sed -i -e 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config";
+    # See "Note: Relabeling your system after switching from the disabled mode" section on:
+    # https://documentation.suse.com/sle-micro/5.4/html/SLE-Micro-all/cha-selinux-slemicro.html
+    assert_script_run "touch /etc/selinux/.autorelabel";
+    process_reboot(trigger => 1);
+    check_disabled;
+    trup_call('setup-selinux');
```

Comment:
do you also need to do some additional checks in PC case as above? e.g. `assert_script_run("semanage port -a -t ssh_port_t -p tcp $upload_port");`?

Reply:
> do you also need to do some additional checks in PC case as above? e.g. `assert_script_run("semanage port -a -t ssh_port_t -p tcp $upload_port");`?

I believe the whole `if (is_public_cloud) {` section should be moved out of  the `# install and enable SELinux if not done by default` code block.  @grisu48 WDYT?

Reply:
If you find a better suiting spot, yes. But where else would you do this?

Context: This is something public_cloud specific, as this is the only place where we need this tunnel. There it is necessary, everywhere else it isn't.

Reply:
question is, do we need to do it again or first time is enough?

Reply:
I **think** one time should be enough, although subsequent runs won't hurt either.